### ISSUE TITLE
Central Database of Spool Weights by Manufacturer

### ISSUE THREAD

**Issue by cjohnson-phys** [Donkie/Spoolman#33]
First off, this is exceptional work and since I have been watching there has been a remarkable explosion of ideas.

I would like to add one more that I think would help the project.

What about a database of spool weights by manufacturer? It would be great if it were a central database that could accept submissions by users (or ideally manufacturers) for empty spool weights. Then Spoolman could update from that website and not have the user need an empty spool to know what the approximate empty spool weight is.

But I can understand having a central database for all users is quite an undertaking so would it be a good idea to have a dedicated local table that stores spool weights by manufacturer and type? For example:
Sparta3D | Cardboard Spool | 100g
Sparta3D | Clear Plastic Spool | 80g
Fusion Filaments | Plastic Spool | 85g
Fusion Filaments | Refill Spool | 91g
etc...   (these numbers are totally made up, but I do have these empty spools so I could provide these details for future users)

There would be an added benefit of a pre-made list of manufacturers that could populate a drop-down menu when creating new filaments in Spoolman

**Comment by Donkie** [comment]
Yes this is something we could definitely streamline more. You have some good ideas. I will think about this :)

**Comment by cjohnson-phys** [comment]
If this is something you think could fit in, then I'd bet we could setup a survey on discord to see what empty spools people have and what they weigh. Get a solid sample and average the duplicates.

I'd be happy to help the cause if you need it

**Comment by Ktmdude88** [comment]
There is several databases of empty spool weights floating around might have to compile them though. 

Examples
https://www.onlyspoolz.com/portfolio/
https://spoolweight.is/the-big-list/

https://3dprintingwiki.mywikis.wiki/wiki/Spool_weight
https://help.matterhackers.com/article/129-empty-spool-weights

**Comment by cjohnson-phys** [comment]
This is great stuff! I wonder if one of them would be willing to enable an API.

They all seem scrapable though.
